public/js/graph/setup.js

```diff
@@ -2,13 +2,17 @@
 // Globals
 var timetable = 'static/res/timetable.html';
 var clickedCourses = [];
-var FCEs = 0;
+
+// FCEs for current graph
+var FCEs = 0; 
```

Comment:
We could leave `totalFCEs` as the only global variable and just increase the credit count with that variable. Do you think we have any need of keeping track of credit counts for individual graphs? Right now I store those as cookies, but if not I could make it simple and remove that code, and keep `totalFCEs` as the only global. It's really the only reason `FCEs` is still global.